src/main/cpp/chassis/commands/season_specific_commands/DriveToBranch.cpp

```diff
@@ -0,0 +1,63 @@
+//====================================================================================================================================================
+// Copyright 2025 Lake Orion Robotics FIRST Team 302
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+// OR OTHER DEALINGS IN THE SOFTWARE.
+//====================================================================================================================================================
+#include "chassis/commands/season_specific_commands/DriveToBranch.h"
+#include "fielddata/ReefHelper.h"
+#include "fielddata/FieldConstants.h"
+
+DriveToBranch::DriveToBranch(subsystems::CommandSwerveDrivetrain *chassis, SwerveContainer::BranchLocation location)
+    : DriveToAprilTagTarget(chassis),
+      m_location(location)
+{
+}
+
+void DriveToBranch::Initialize()
+{
+    auto reefHelper = ReefHelper::GetInstance();
+    auto taginfo = reefHelper->GetNearestReefTag();
+
+    if (taginfo.has_value())
+    {
+        auto tag = taginfo.value();
+        if (m_location == SwerveContainer::BranchLocation::RIGHT_BRANCH)
+        {
+            auto rightBranch = ReefHelper::GetInstance()->GetNearestRightReefBranch(tag);
```

Comment:
Do we need reef helper or could that just be put here?

Reply:
I do think a lot of Reef helper code could go here, but I do like having these helper classes because it makes the code easier to read, but also it is necessary for mechanisms and autons